ament_cmake_pclint/package.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>ament_cmake_pclint</name>
+  <version>0.0.3</version>
```

Comment:
The version should match the other packages in this repo which are currently 0.4.0.

Same in the other package as well as the `setup.py` file.

Reply:
Done.